test/fixtures/commits_after_ci.json

```diff
@@ -0,0 +1,16 @@
+  {
+    "commits": [{
+      "commit": {
+        "committedDate": "2017-10-25T11:27:02Z",
+        "oid": "6c0945cbeea2cbbc97d13a3d9e3fe68bd145b985",
+        "message": "fixup: adjust spelling",
+        "author": {
+          "login": "bar"
+        }
+      }
+    }],
+    "comment": [{
+      "bodyText": "CI: https://ci.nodejs.org/job/node-test-pull-request/10984/",
+      "publishedAt": "2017-10-24T11:19:25Z"
+    }]
+  }
```

Comment:
Could you add a new line here? (We probably should have a lint rule for this.)

Reply:
@apapirovski just so you know every other `.json` files does not have EOL.

Reply:
@buji1993 This still needs a new line. Thanks.